frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/basicTicketImage/basicTicketImage.component.tsx

```diff
@@ -0,0 +1,158 @@
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { useContext, useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import EmptyImageIcon from '@assets/icons/outlined/add_image_thin-outlined.svg';
+import BrokenImageIcon from '@assets/icons/outlined/broken-outlined.svg';
+import EnlargeImageIcon from '@assets/icons/outlined/enlarge_image-outlined.svg';
+import { HiddenImageUploader } from '@controls/hiddenImageUploader/hiddenImageUploader.component';
+import { ProjectsHooksSelectors } from '@/v5/services/selectorsHooks/projectsSelectors.hooks';
+import { formatMessage } from '@/v5/services/intl';
+import { validateImg } from '@controls/hiddenImageUploader/Image.helper';
+import {
+	ActionsList,
+	ActionsSide,
+	BrokenImageContainer,
+	Container,
+	EmptyImageContainer,
+	EnlargeContainer,
+	IconText,
+	Image,
+	ImageSide,
+	OverlappingContainer,
+	PropertyName,
+} from './basicTicketImage.styles';
+import { TicketImageActionContext } from './actions/ticketImageActionContext';
+
+const TicketImage = ({ imgIsInvalid }) => {
+	const { isAdmin } = ProjectsHooksSelectors.selectCurrentProjectDetails();
+
+	const { imgSrc, setImgSrc } = useContext(TicketImageActionContext);
+
+	if (!imgSrc && !imgIsInvalid) {
+		return (
+			<HiddenImageUploader
+				onChange={setImgSrc}
+				disabled={!isAdmin}
+			>
+				<EmptyImageContainer disabled={!isAdmin}>
+					<EmptyImageIcon />
+					<IconText>
+						<FormattedMessage id="viewer.cards.ticketImage.addImage" defaultMessage="Add Image" />
+					</IconText>
+				</EmptyImageContainer>
+			</HiddenImageUploader>
+		);
+	}
+
+	if (!imgSrc && imgIsInvalid) {
+		return (
+			<HiddenImageUploader
+				onChange={setImgSrc}
+				disabled={!isAdmin}
+			>
+				<BrokenImageContainer disabled={!isAdmin}>
+					<BrokenImageIcon />
+					<IconText>
+						<FormattedMessage id="viewer.cards.ticketImage.unsupportedImage" defaultMessage="Unsupported Image" />
+					</IconText>
+				</BrokenImageContainer>
+			</HiddenImageUploader>
+		);
+	}
+
+	return (
+		<OverlappingContainer>
+			<Image
+				src={imgSrc}
+				alt={formatMessage({ id: 'viewer.cards.ticketImage.image', defaultMessage: 'image' })}
+				onChange={setImgSrc}
+			/>
+			<EnlargeContainer>
+				<EnlargeImageIcon />
+				<IconText>
+					<FormattedMessage id="viewer.cards.ticketImage.enlarge" defaultMessage="Enlarge" />
+				</IconText>
+			</EnlargeContainer>
+		</OverlappingContainer>
+	);
+};
+
+type BasicTicketImageProps = {
+	imgSrc?: string,
+	viewpoint?: any,
+	title: string,
+	className?: string,
+	onChange?: ({ imgSrc, viewpoint }) => void;
+	children: any,
+};
+export const BasicTicketImage = ({
+	children,
+	imgSrc: inputImgSrc,
+	viewpoint:
+	inputViewpoint,
+	title,
+	className,
+	onChange,
+}: BasicTicketImageProps) => {
+	const [viewpoint, setViewpoint] = useState(inputViewpoint);
+	const [imgSrc, setImgSrcUnsafe] = useState(inputImgSrc);
```

Comment:
To distinguish it from the `setImgSrc` passed to the context. `setImgSrcUnsafe` only sets the `imgSrc`, but does not validate it, meaning calling it directly may allow other components to set invalid image sources. This way I pass to the context the richer `setImgSrc` so that components accessing the context see a more consistent naming and they do not need to revalidate the `imgSrc` directly as that is handled in basicTicketImage only.